wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/registry/LanguageServerFileWatcher.java

```diff
@@ -62,6 +66,8 @@ private void send(LanguageServer server, String filePath, FileChangeType changeT
     DidChangeWatchedFilesParams params =
         new DidChangeWatchedFilesParams(
             Collections.singletonList(new FileEvent(prefixURI(filePath), changeType)));
+
+    LOG.debug("sending " + changeType + " to LS: " + filePath);
```

Comment:
better to use parameterized logging `LOG.debug("sending {} to LS: {}", changeType, filePath);`